### PR TITLE
Updates TD save code to use user data folders.

### PR DESCRIPTION
**Pull request by OmniBlade**

### CODE REVIEW

File: tiberiandawn/loaddlg.cpp [hunk 1]
```diff
@@ -481,6 +482,22 @@ int LoadOptionsClass::Process(void)
...
+                    */
+                    if (game_descr[0] == '(') {
+                        char* ptr = strchr(game_descr, ')');
+                        if (ptr != NULL) {
+                            char* dst = game_descr;
+                            const char* src = ptr + 1; // Skip ')'
+
+                            // Don't use strcpy, as the memory regions overlap.
+                            while ((*dst++ = *src++) != '\0')
+                                ; // Copy string
```
Comment (giulianobelinassi) [inline tiberiandawn/loaddlg.cpp:497]: This code resembles some code written by me (redalert/loaddlg.cpp:561).  Please refactor that into a common function.

**Review by giulianobelinassi — CHANGES_REQUESTED** [review]

File: tiberiandawn/loaddlg.cpp (hunk 1 above)
Comment (OmniBlade) [reply tiberiandawn/loaddlg.cpp:497]: I copy/pasted it from the RA load dialog as I assumed it fixed a bug you came upon in your DSi porting.

File: redalert/conquer.cpp [hunk 2]
```diff
@@ -4256,7 +4256,7 @@ bool Force_CD_Available(int cd)
 #endif
 
 #ifdef FRENCH
-                sprintf(buffer, "Ins�rez le %s", _cd_name[cd]);
+                sprintf(buffer, "Ins�rez le %s", _cd_name[cd]);
```
Comment (tomsons26) [inline redalert/conquer.cpp:4259]: mmm?

Comment (OmniBlade) [reply redalert/conquer.cpp:4259]: Fixes an accidental unicode change from 6008d34bedc5f4fd445ad1a078c5fb1fa6cee3d2.

**Review by giulianobelinassi — APPROVED** [review]

**Pull request merged**